src/packetizer.iced

```diff
@@ -93,7 +93,7 @@ exports.Packetizer = class Packetizer
     # header. If we can't get that much, we'll just have to wait!
     return @WAIT unless (f_full = @_ring.grab frame_len)?
 
-    [w,r] = unpack f_full
+    [w,r] = unpack f_full.slice 0, frame_len
```

Comment:
do you want to do this always?

Reply:
i think so, going to be testing on the server